src/main/java/com/dev/cinema/model/ShoppingCart.java

```diff
@@ -0,0 +1,58 @@
+package com.dev.cinema.model;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_cart")
```

Comment:
```suggestion
@Table(name = "shopping_carts")
```

Reply:
see checklist

Reply:
Attention: you can have some problem with naming of the table. Let’s use orders instead of order. You can do that by using annotation @Table(name = “orders”).

Reply:
as far as i understand, "order" is a keyword in hql, but "shopping_cart" is not